src/platforms/react-native/migration.mdx

```diff
@@ -4,6 +4,49 @@ sidebar_order: 100
 description: "Learn about migrating from 1.x to 2.x to enable release health tracking and native stack traces by default."
 ---
 
+## From 4.x to 5.x
+
+The React Native SDK version 5 supports both Legacy (from RN 0.65 and above) and New Architecture (from RN 0.69 and above) as well as the new React Native Gradle Plugin (introduced in RN 0.71).
+
+### Naming and behavior changes
+
+Rename the following `Sentry.init` options:
+- `enableAutoPerformanceTracking` to `enableAutoPerformanceTracing`
+- `enableOutOfMemoryTracking` to `enableWatchdogTerminationTracking`
+- `ReactNativeTracingOptions.idleTimeout` to `ReactNativeTracingOptions.idleTimeoutMs`
+- `ReactNativeTracingOptions.maxTransactionDuration` to `ReactNativeTracingOptions.finalTimeoutMs`
+
+If you are using `TouchEventBoundary` and `accessibilityLabel` set the following:
+```javascript
+Sentry.wrap(App, { touchEventBoundaryProps: { labelName: 'accessibilityLabel' } })
+```
+
+Current stack trace for `Sentry.captureMessage` has been moved from `event.exception` to `event.threads`.
+
+Default `breadcrumb.level` is `info`, value `critical` has been removed and `log` is transformed to `debug` on iOS and Android.
+
+Bundled `sentry-cli` version has been bumped to [v2.10.0](https://github.com/getsentry/sentry-cli/releases/tag/2.10.0).
+
+### iOS specific changes
+
+The following metadata are sync from `sentry-cocoa` to `react-native`:
+- `tags`
+- `extra`
+- `fingerprint`
+- `level`
+- `environment`
+- `breadcrumbs`
+
+The synchronized `fingerprint` might affect grouping of issues in Sentry.
```

Comment:
`l`: Is there any additional info we can provide? A link?

Reply:
We could add:

React Native `event.fingerprint` is merged with iOS `fingerprint` without duplicates with RN items being first in the order.

Reply:
Yes that's good info IMO